src/main/java/org/usfirst/frc4904/robot/subsystems/FourBarLinkage.java

```diff
@@ -0,0 +1,27 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc4904.robot.commands.FourBarLeverUp;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+public class FourBarLinkage extends Subsystem {
+	public static final DoubleSolenoid.Value UP = DoubleSolenoid.Value.kForward;
+	public static final DoubleSolenoid.Value DOWN = DoubleSolenoid.Value.kReverse;
+	protected final DoubleSolenoid lever;
+
+	public FourBarLinkage(DoubleSolenoid lever) {
+		super("FourBarLinkage");
+		this.lever = lever;
+	}
+
+	public void set(boolean up) {
+		if (up) this.lever.set(UP);
+		else this.lever.set(DOWN);
+	}
+
+	@Override
+	protected void initDefaultCommand() {
+		setDefaultCommand(new FourBarLeverDown());
```

Comment:
We could just go with no default command, as not to jolt the robot as soon as we initialize it, but this works maybe